### ISSUE TITLE
Unable to get ENR information from Magi

### ISSUE THREAD

**Issue by barnabasbusa** [a16z/magi#247]
I'm trying to get the enr info off of magi using the same API call that I'm using with op-node.

(json rpc call for `opp2p_self`). 

Does Magi have API parity with op-node? I couldn't find any docs about available api endpoints for it.

**Comment by ncitron** [comment]
Yes we don't have that API endpoint implemented yet. We should document which one's we're missing, and work on implementing them, as I'm pretty sure they are all relatively simple.

**Comment by alessandromazza98** [comment]
> Yes we don't have that API endpoint implemented yet. We should document which one's we're missing, and work on implementing them, as I'm pretty sure they are all relatively simple.

Hei @ncitron I'd be interested in doing that. Is there any api already implemented I could see in magi in order to take inspiration ?

**Comment by ncitron** [comment]
That would be great @alessandromazza98! You can take a look at how we're implementing API routes [here](https://github.com/a16z/magi/blob/master/src/rpc/mod.rs).

**Comment by barnabasbusa** [comment]
Any update on this issue?

**Comment by ncitron** [comment]
@alessandromazza98 are you still interested in working on this? If not I can look into this at some point.
